File: Basics/Basics1.cs
using System;
public class Basic
{
    public static void Main(string[] args)
    {
        Console.WriteLine("Helllooo c#..");

        Console.WriteLine("Enter the Name..");
        string name = Console.ReadLine();

        Console.WriteLine("Enter the Age..");
        int.TryParse(Console.ReadLine(),out int age);

        Console.WriteLine("Enter the City..");
        string city = Console.ReadLine();

        Console.WriteLine("{0} {1} {2}",name,age,city);

        Console.WriteLine("Enter the n:");
        int.TryParse(Console.ReadLine(),out int n);
        for(int i=1;i<=n;i++)
        {
            for(int j=1;j<=i;j++)
            {
                Console.Write("*");
            }
            Console.WriteLine();
        }

    }
}

File: PatternsLLM/Basics/Basics1.java
import java.util.Scanner;

public class Basics1 {
    
    public static void main(String[] args) {
        System.out.println("Helloo World");

        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter the name: ");
        String name = sc.nextLine();

        System.out.print("Enter the age: ");
        int age = Integer.parseInt(sc.nextLine());
        

        System.out.print("Enter the city: ");
        String city = sc.nextLine();

        System.out.printf("Name: %s Age: %d City: %s\n", name, age, city);



        System.out.print("Enter the n:");
        int n = Integer.parseInt(sc.nextLine());

        for(int i=1;i<=n;i++)
        {
            for(int j=1;j<=i;j++)
            {
                System.out.print("*\t");
            }
            System.out.println();
        }
    }

}